README.md

```diff
@@ -74,6 +77,9 @@ giniConfiguration.backgroundColor = UIColor.white
 GiniVision.setConfiguration(giniConfiguration)
 ```
 
+## iPad support
+On iPad, unlike the iPhone, the Gini Vision Library UI supports both landscape and portrait orientations, full rotation being enabled by default in both APIs (*Screen* and *Component*). So in case you don't want any orientation, you need to disable it in your project settings (*Project Settings &rarr; General &rarr; Deployment Info &rarr; Device orientation*). 
```

Comment:
So in case you don't want ~~any~~ a particular orientation

Reply:
You should also list the higher resolution images that are required for iPads.